npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker.rb

```diff
@@ -340,7 +340,8 @@ def build_updated_dependency(update_details)
       def latest_resolvable_transitive_security_fix_version_with_no_unlock
         versions = T.let([], T::Array[Gem::Version])
 
-        versions.push(T.must(latest_released_version)) if latest_released_version
+        latest_released_ver = latest_released_version
```

Comment:
Personally, I'd avoid the variable and just use latest_released_version

Reply:
Yeah, I agree. I'm not sure what the error this is fixing is. The PR description tells me nothing.